content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md

```diff
@@ -0,0 +1,939 @@
+---
+api_metadata:
+  apiVersion: "discovery.k8s.io/v1"
+  import: "k8s.io/api/discovery/v1"
+  kind: "EndpointSlice"
+content_type: "api_reference"
+description: "EndpointSlice 是实现某 Service 的端点的子集."
+title: "EndpointSlice"
+weight: 3
+auto_generated: true
+---
+
+<!--
+description: "EndpointSlice represents a subset of the endpoints that implement a service."
+-->
+
+`apiVersion: discovery.k8s.io/v1`
+
+`import "k8s.io/api/discovery/v1"`
+
+## EndpointSlice {#EndpointSlice}
+
+<!--
+EndpointSlice represents a subset of the endpoints that implement a service. For a given service there may be multiple EndpointSlice objects, selected by labels, which must be joined to produce the full set of endpoints.
+-->
+EndpointSlice 是实现某 Service 的端点的子集。
+一个 Service 可以有多个 EndpointSlice 对象与之对应，
+必须将所有的 EndpointSlice 拼接起来才能形成一套完整的端点集合。
+Service 通过标签来选择 EndpointSlice。
+
+<hr>
+
+- **apiVersion**: discovery.k8s.io/v1
+
+- **kind**: EndpointSlice
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata.
+  -->
+  标准的对象元数据。
+
+- **addressType** (string), <!--required-->必需
+  
+  <!--
+  addressType specifies the type of address carried by this EndpointSlice. All addresses in this slice must be the same type. This field is immutable after creation.
+  -->
+  addressType 指定当前 EndpointSlice 携带的地址类型。
+  一个 EndpointSlice 只能携带同一类型的地址。
+  EndpointSlice 对象创建完成后不可以再更改 addressType 字段。
+  
+  <!--
+  The following address types are currently supported: * IPv4: Represents an IPv4 Address. * IPv6: Represents an IPv6 Address. * FQDN: Represents a Fully Qualified Domain Name.
+  -->  
+  目前支持的地址类型为：
+  * IPv4：表示 IPv4 地址。
+  * IPv6：表示 IPv6 地址。
+  * FQDN：表示完全限定域名。
+
+- **endpoints** ([]Endpoint), <!--required-->必需
+
+  <!--
+  *Atomic: will be replaced during a merge*
+  -->
+  **原子性：合并期间将被替换**
+
+  <!--
+  endpoints is a list of unique endpoints in this slice. Each slice may include a maximum of 1000 endpoints.
+  -->
+  endpoints 是当前 EndpointSlice 中一组唯一的端点。
+  每个 EndpointSlice 最多可以包含 1000 个端点。
+
+  <a name="Endpoint"></a>
+
+  <!--
+  *Endpoint represents a single logical "backend" implementing a service.*
+  -->
+  **端点是实现某 Service 的一个逻辑“后端”。**
+
+  - **endpoints.addresses** ([]string), <!--required-->必需
+    
+    <!--
+    *Set: unique values will be kept during a merge*
+    -->
+    **集合：不重复的值在合并期间会被保留**

```

Comment:
May need to add a blank before line 86. Similar issues exist when preview: 
https://deploy-preview-34928--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/